system/Database/MySQLi/PreparedQuery.php

```diff
@@ -34,7 +35,7 @@ class PreparedQuery extends BasePreparedQuery
      * @param array $options Passed to the connection's prepare statement.
      *                       Unused in the MySQLi driver.
      *
-     * @return mixed
+     * @return static
```

Comment:
```suggestion
     * @return $this
```

Reply:
I will revert this, following changes in #6750